container/image.bzl

```
@@ -459,8 +462,10 @@ def _impl(
             null_cmd = null_cmd,
         )
 
-    # Construct a temporary name based on the build target.
-    tag_name = "{}:{}".format(_repository_name(ctx), name)
+    # Construct a temporary name based on the build target. This is the name
+    # of the docker container.
+    final_tag = tag_name if tag_name else name
+    tag_name = "{}:{}".format(_repository_name(ctx), final_tag)
```

```suggestion
    if not tag_name:
        tage_name = "{}:{}".format(_repository_name(ctx), name)
```

Can we do this to allow users to specific the whole tag, not just part of it?

---

I don't think it's a good idea:
1. It already provides `repository`, to override the default `bazel/`. We can't delete that.
2. It is non-trivial to generate the string between `bazel/` and `:` if, say, the user wants to change only the tag (our case).